pixelgl/input.go

```diff
@@ -431,6 +435,15 @@ func (w *Window) doUpdateInput() {
 	w.prevInp = w.currInp
 	w.currInp = w.tempInp
 
+	w.pressEvents = w.tempPressEvents
+	w.releaseEvents = w.tempReleaseEvents
+
+	// Clear last frame's events
+	for i := range w.tempPressEvents {
+		w.tempPressEvents[i] = false
+		w.tempReleaseEvents[i] = false
+	}
+
 	w.tempInp.repeat = [KeyLast + 1]bool{}
```

Comment:
I'm not sure which is faster or if it matters, but might consider doing for temp*Events what is done for `w.tempInp.repeat` (line 447), would at least be consistent.

```golang
w.tempPressEvents = [KeyLast + 1]bool{}
w.tempReleaseEvents = [KeyLast +1]bool{}
```

Reply:
Hi, Thanks for the feedback. Good suggestion - I like the ```w.tempPressEvents = [KeyLast + 1]bool{}``` notation better, I'm not sure why I missed that the first time around.